From: YOKOTA Hiroshi <yokota.hgml@gmail.com>
Date: Sun, 18 Feb 2024 23:14:31 +0900
Subject: Fix book store chooser error

Reproducer:
1. "Get Books" -> "Choose Stores" from tool button bar

----
Traceback (most recent call last):
  File "/usr/lib/calibre/calibre/gui2/actions/store.py", line 116, in choose
    d = StoreChooserDialog(self.gui)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/calibre/calibre/gui2/store/config/chooser/chooser_dialog.py", line 21, in __init__
    self.config_widget = StoreChooserWidget()
                         ^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/calibre/calibre/gui2/store/config/chooser/chooser_widget.py", line 15, in __init__
    self.setupUi(self)
  File "/usr/lib/calibre/calibre/gui2/store/config/chooser/chooser_widget_ui.py", line 35, in setupUi
    self.results_view = ResultsView(parent=Form)
                        ^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: ResultsView.__init__() got an unexpected keyword argument 'parent'
----
---
 src/calibre/gui2/store/config/chooser/results_view.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/calibre/gui2/store/config/chooser/results_view.py b/src/calibre/gui2/store/config/chooser/results_view.py
index c098a3d..3eafc6c 100644
--- a/src/calibre/gui2/store/config/chooser/results_view.py
+++ b/src/calibre/gui2/store/config/chooser/results_view.py
@@ -13,8 +13,8 @@ from calibre.gui2.store.config.chooser.models import Matches, Delegate
 
 class ResultsView(QTreeView):
 
-    def __init__(self, *args):
-        QTreeView.__init__(self,*args)
+    def __init__(self, parent=None):
+        QTreeView.__init__(self,parent)
 
         self._model = Matches([p for p in store_plugins()])
         self.setModel(self._model)
